Source/Fuse.Elements/Resources/MultiDensityImageSource.uno

```diff
@@ -92,6 +110,7 @@ namespace Fuse.Resources
 			}
 
 			SwapActive(use);
+			OnSourcesChanged();
```

Comment:
The name of the event seems wrong, shouldn't it be `OnActiveChanged`?  It probably belongs in the `SwapActive` function.